packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -239,8 +240,7 @@ export const messages = {
     'We couldn’t push your source due to conflicts. Make sure that you want to overwrite the metadata in your org with your local files, then run "SFDX: Push Source to Default Scratch Org and Override Conflicts".',
   error_no_default_username:
     'No default org is set. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org" to set one.',
-  error_no_default_devhubusername:
-    'No default Dev Hub is set. Run "SFDX: Authorize a Dev Hub" to set one.',
+  error_no_default_devhubusername: 'No default Dev Hub is set.',
```

Comment:
We may want to keep this message the way it was. Here it is in action:

![Screen Shot 2019-06-07 at 10 51 43 AM](https://user-images.githubusercontent.com/10779873/59123545-4e71ac80-8912-11e9-95a1-71c6fbdcb947.png)